end-to-end-tests/tests/telemetry/errors.spec.ts

```diff
@@ -26,54 +26,68 @@ async function waitForBackgroundPageRequest(
   return offscreenPage?.waitForRequest(errorServiceEndpoint);
 }
 
+async function getSentErrors(
+  extensionId: string,
+  context: BrowserContext,
+  errorServiceEndpoint: "https://browser-intake-datadoghq.com/api/v2/*",
+) {
+  // TODO: due to Datadog SDK implementation, it will take ~30 seconds for the
+  //  request to be sent. We should figure out a way to induce the request to be sent sooner.
+  //  See this datadog support request: https://help.datadoghq.com/hc/en-us/requests/1754158
+  const request = await waitForBackgroundPageRequest(
+    context,
+    extensionId,
+    errorServiceEndpoint,
+  );
+
+  return request
+    ?.postData()
+    ?.split("\n")
+    .map((log) => JSON.parse(log));
+}
+
 test.use({
   additionalRequiredEnvVariables: [
     "DATADOG_CLIENT_TOKEN",
     "DEV_EVENT_TELEMETRY",
   ],
 });
 
-test("can report application error to telemetry service", async ({
+test("can report extension console error to telemetry service", async ({
```

Comment:
It's not exclusive to extension console, "application" wasn't meant to refer to the pixiebrix-app project, if that's why you removed it.
```suggestion
test("can report errors to telemetry service", async ({
```

Reply:
this was to cover errors that are reported from the extension console page, vs in the background service worker